src/client/app/containers/ExportContainer.ts

```diff
@@ -15,8 +15,8 @@ function transformLineReadingToLegacy(reading: CompressedLineReading): [number,
 	return [reading.startTimestamp, reading.reading];
 }
 
-function transformBarReadingToLegacy(reading: CompressedBarReading): [number, number] {
-	return [reading.startTimestamp, reading.reading];
+function transformBarReadingToLegacy(reading: CompressedBarReading): [number, number, number] {
```

Comment:
All these changes seem to be from PR #668. They either should be removed or we need to wait until that clears before we merge this.